ui/apps/platform/src/utils/searchUtils.ts

```diff
@@ -199,6 +199,6 @@ export function getListQueryParams(
                 sortOption,
             },
         },
-        { arrayFormat: 'repeat', allowDots: true }
```

Comment:
This was removed since there is no way an array of items could be passed to the `qs.stringify` function. Array items that are part of the SearchFilter are processed internally first.